Makefile

```diff
@@ -110,7 +110,10 @@ secrets-update:
 	rm secrets.tar.gz
 
 e2e:
-	go test ./test/e2e -timeout 60m -v -ginkgo.v -tags e2e
+	go test ./test/e2e -tags e2e -c -o e2e.test
+
+test-e2e: e2e
```

Comment:
How about calling the previous target e2e.test, given that's the name of the binary it actually creates?  Avoids changing the meaning of `make e2e`.

Reply:
Yeah, I thought about it, but I think it is going to be confusing to have both `e2e` and `e2e.test`. Also if you were to see these targets for the first time, which one is actually running tests? I think it makes sense to standartise on `test-*` for targets that run tests (we already have `test-go`, `test-python`). I'll just send an email to the team about changes in the target.

Reply:
`test-e2e` makes sense, I agree. Still tempted to call the previous target e2e.test as that's the file it creates.